### PR TITLE
Added Serverless ReactJS Universal Rendering Boilerplate 

### PR DESCRIPTION
**Pull request by TylorShin**
Hi.
I made the boilerplate example about [Isomorphic rendering ReactJS in Serverless].
It's still in progress, but I think some people can get help from this example.

Thanks.

### CODE REVIEW

File: community-examples.json [hunk 1]
```diff
@@ -192,6 +192,10 @@
     "name": "serverless-modern-koa",
     "description": "Serverless modern koa starter kit",
     "githubUrl": "https://github.com/barczaG/serverless-modern-koa"
+  },
+  {
+    "name": "Serverless ReactJS Universal Rendering Boilerplate",
+    "description": "ReactJS webapp does universal(isomorphic) rendering with Serverless starter kit",
```
Comment (christophgysin) [inline community-examples.json]: There seems to be a missing space after "universal".

File: community-examples.json [hunk 2]
```diff
@@ -192,6 +192,10 @@
...
+    "githubUrl": "http://github.com/TylorShin"
```
Comment (christophgysin) [inline community-examples.json]: Could you link directly to the repository?

File: community-examples.json (hunk 1 above)
Comment (TylorShin) [inline community-examples.json]: Oh. I had a mistake. I changed it!

File: community-examples.json (hunk 2 above)
Comment (TylorShin) [inline community-examples.json]: Okay.
It was my mistake. I fixed it.

**Pull request merged**